CRM/CiviAwards/Service/ApplicantManagementAwardDetailPostProcessor.php

```diff
@@ -95,7 +95,7 @@ public function processCaseTypeCustomGroupsOnUpdate(AwardDetail $awardDetail) {
       }
     }
 
-    if ($this->awardSubTypeChanged($awardDetail)) {
+    if ($this->awardSubtypeChanged($awardDetail)) {
```

Comment:
Not sure why we changed this function name? The name is already okay?

Reply:
Same as below,
> Since now we have Subtype throughout the application, I refactored this to follow same convention. Maybe we can keep it?